backend/src/schema/types/type/Image.gql

```diff
@@ -1,10 +1,5 @@
 type Image {
-  url: ID!,
-  # urlW34: String,
-  # urlW160: String,
-  # urlW320: String,
-  # urlW640: String,
-  # urlW1024: String,
+  url(width: Int): ID!,
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-01T06:43:34Z" title="Wednesday, April 1st 2020, 8:43:34 am +02:00">Apr 1, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,10 +1,5 @@
 type Image {
-  url: ID!,
-  # urlW34: String,
-  # urlW160: String,
-  # urlW320: String,
-  # urlW640: String,
-  # urlW1024: String,
+  url(width: Int): ID!,
```


I like this a lot!!